sdk/identity/identity/test/manual/README.md

```diff
@@ -26,27 +26,17 @@ You will need to configure an AAD App Registration as follows:
 - Click on the app that you want to use to authenticate.
 - Go to the Authentication tab of your AAD application.
 - Click on `+ Add a platform`, select `Single-page application`, enter `http://localhost:8080` as the redirect URI, then make sure to include implicit grant for "Access tokens" and "ID tokens".
-- Keep in mind that if you belong to an organization, other restrictions based on the organization configurations might prevent you from authenticating. If These steps don't end up being effective, try again on a personal account.
+- Go to the `API permissions` tab of your AAD application. Click on `Add a permission`, then go to `APIs my organization uses` and search for `Microsoft.ServiceBus`, then add this permission.
+- Keep in mind that if you belong to an organization, other restrictions based on the organization configurations might prevent you from authenticating. If these steps don't end up being effective, try again on a personal account.
 
-Grant access to this AAD application to your Key Vault by:
+Grant access to this AAD application to your Service Bus by:
 
-- Creating a Key Vault (if you haven't created one).
-- Either in the "Access policies" section of the creation form, or by going to your Key Vault's "Access policies" page, click con `+ Add Access Policy`, select all permissions, then select your AAD application as the "principal", then click "Add", then click "Save" if applicable.
+- Creating a Service Bus namespace (if you haven't created one).
+- Either in the "Access policies" section of the creation form, or by going to the Service Bus namespace's "Access policies" page, click con `+ Add Access Policy`, select all permissions, then select your AAD application as the "Service Bus Data Owner", then click "Add", then click "Save" if applicable.
```

Comment:
Just checked.. you can't add access policies while creation.. you need to go to the namespace and do it.

Reply:
Thank you Harsha!

Reply:
```suggestion
- Go to the Service Bus namespace's "Access control (IAM)" tab, click on `+ Add` > `Add role assignment`, Pick "Service Bus Data Owner" role, and select your AAD application to assign the role, then click "Save".
```